src/flp/types/fixedpoint_l2.rs

```diff
@@ -172,22 +195,25 @@ use std::{convert::TryFrom, convert::TryInto, fmt::Debug, marker::PhantomData};
 /// particular, exactly the following types are supported:
 /// `FixedI16<U15>`, `FixedI32<U31>` and `FixedI64<U63>`.
 ///
+/// The type provides an [`add_noise`] function that adds discrete Gaussian noise to an
```

Comment:
```suggestion
/// The type provides an [`add_noise_to_result`] function that adds discrete Gaussian noise to an
```

Reply:
Rewritten to:
```
/// The type implements the [`TypeWithNoise`] trait. The [`add_noise_to_result`] function adds
/// discrete Gaussian noise to an aggregate share, calibrated to the passed privacy budget.
/// This will result in the aggregate satisfying zero-concentrated differential privacy.
```